Examples/Algorithms/Fatras/src/FatrasAlgorithm.cpp

```diff
@@ -68,14 +68,14 @@ namespace {
 // This always uses the EigenStepper with default extensions for charged
 // particle propagation and is thus limited to propagation in vacuum at the
 // moment.
-template <typename magnetic_field_t>
+// @TODO: Remove this, unneeded after #675
```

Comment:
Isn't this #675 ? So it's unneeded now, or in a follow-up PR?

Reply:
Yeah, what I meant, strictly speaking this is not necessary with the changes in this PR. However, I didn't want to make it larger than it is, so postpone this to a follow-up PR. This TODO is just to keep a reference to why this is more complicated than it needs to be.